lib/cdo/contact_rollups/v2/contact_rollups.rb

```diff
@@ -1,12 +1,12 @@
 class ContactRollupsV2
-  def self.build_contact_rollups(log_collector)
+  def self.build_contact_rollups(log_collector, pardot_steps=false)
```

Comment:
is the idea here that we are only going to do the full pardot steps very rarely, so we will do a code change to enable it? Nothing is calling this with true now right?

Reply:
@hacodeorg may object -- was just a convenience way for me to run the non-Pardot steps. I think it might be a good idea generally though to have to explicit about running the Pardot steps -- we both have Pardot API keys in our locals.yml, so it doesn't seem implausible that we might accidentally start writing to Pardot. Some of the data in Pardot didn't come from our production DB and has no backup as far as I know, so I'm in favor of putting guardrails up.